staging/src/k8s.io/apiserver/pkg/admission/plugin/resourcequota/admission.go

```diff
@@ -110,11 +116,21 @@ func (a *QuotaAdmission) SetExternalKubeInformerFactory(f informers.SharedInform
 // SetQuotaConfiguration assigns and initializes configuration and evaluator for QuotaAdmission
 func (a *QuotaAdmission) SetQuotaConfiguration(c quota.Configuration) {
 	a.quotaConfiguration = c
+	if a.stopCh == nil {
+		a.initializationErr = fmt.Errorf("quota configuration configured between stop channel")
+		return
+	}
 	a.evaluator = NewQuotaEvaluator(a.quotaAccessor, a.quotaConfiguration.IgnoredResources(), generic.NewRegistry(a.quotaConfiguration.Evaluators()), nil, a.config, a.numEvaluators, a.stopCh)
 }
 
 // ValidateInitialization ensures an authorizer is set.
 func (a *QuotaAdmission) ValidateInitialization() error {
+	if a.initializationErr != nil {
+		return nil
```

Comment:
```suggestion
		return a.initializationErr 
```

Reply:
should return an error, right?

Reply:
dohh..

good catch - also added a test for that